src/components/topic/TopicDirective.js

```diff
@@ -43,15 +43,16 @@ goog.require('ga_topic_service');
               }
             });
 
-            scope.$on('gaTopicChange', function(evt, newTopic) {
-              if (!scope.activeTopic) {
-                scope.topics = gaTopic.getTopics();
-                scope.$applyAsync(function() {
-                  element.find('.ga-topic-item').tooltip({
-                    placement: 'bottom'
-                  });
+            gaTopic.getTopics().then(function(topics) {
+              scope.topics = topics;
```

Comment:
I think you also need to set the `activeTopic` variable otherwise the topic will not be selected in the list, if the `gaTopicChange ` event is triggered before the loading of the directive. I've reproduced once.